cowprotocol/cowswap
# Feature/partial fills - update labels

# Summary

Updating labels to account for partial fills

Orders that are not 100% filled but had partial fills will not use the label `Partial fills`

Also, orders that are almost filled are considered filled using the same logic as the explorer, showing `Filled` instead

# To Test

Not easy to test in the current circumstances

1. Have a order that has partial fills and is closed
* Observe the partial fills label
2. Have an order that has partial fills and is closed and is quite close to 100%
For example, impersonate this account `0x519b70055af55A007110B4Ff99b0eA33071c720a` on `prod` 
* It'll show `Filled` instead of `Expired`
